python/ray/tests/test_memory_scheduling.py

```diff
@@ -0,0 +1,154 @@
+import ray
```

Comment:
import order, all ray imports should go after all other imports (separated by newline)